x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTypesTests.java

```diff
@@ -31,12 +28,11 @@
     private final SqlParser parser = new SqlParser();
 
     private Tuple<Command, SqlSession> sql(String sql) {
-        EsIndex test = new EsIndex("test", TypesTests.loadMapping("mapping-multi-field-with-nested.json", true));
-        Analyzer analyzer = new Analyzer(TestUtils.TEST_CFG, new FunctionRegistry(), IndexResolution.valid(test), null);
-        Command cmd = (Command) analyzer.analyze(parser.createStatement(sql), false);
+        Analyzer analyzer = new Analyzer(new FunctionRegistry(), null);
+        Command cmd = (Command) TestUtils.withContext(TestUtils.TEST_CFG, null, () -> analyzer.analyze(parser.createStatement(sql), false));
```

Comment:
Why is the IndexResolution `null` here? The previous code was loading a json file and created an IndexResolution from it.